access-control-app/int_test/cmr/access_control/int_test/permission_check_test.clj

```diff
@@ -98,8 +98,8 @@
         (= {coll1 permissions}
            (get-collection-permissions user))
         :guest []
-        :registered []
-        "user1" []))
+        :registered ["update" "delete"]
```

Comment:
the users have ingest and delete permissions, but not search and order.  This is because we are saving acl fixtures to access control now.  Ill add a comment